polaris-patterns/.storybook/preview.js

```diff
@@ -70,4 +98,22 @@ export const globalTypes = {
       ],
     },
   },
+  country: {
+    name: 'Country',
+    description: 'Country code',
+    defaultValue: 'CA',
+    toolbar: {
+      icon: 'flag',
+      items: ['CA', 'US', 'DE', 'AU'],
+    },
+  },
+  currency: {
+    name: 'Currency',
+    description: 'Currency code',
+    defaultValue: 'USD',
+    toolbar: {
+      icon: 'credit',
+      items: ['USD', 'CAD', 'AUD', 'GBP', 'EUR', 'JPY', 'CNY', 'DKK'],
```

Comment:
I could add all of [these codes](https://github.com/Shopify/quilt/blob/main/packages/react-i18n/src/currencyCode.ts) in the future, but just for Storybook these few might be enough